src/backend/db/videos.js

```diff
@@ -13,7 +13,22 @@ export const videos = [
     description:
       "At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium voluptatum deleniti atque corrupti quos dolores et quas molestias excepturi sint occaecati cupiditate non provident, similique sunt in culpa qui officia deserunt mollitia animi, id est laborum et dolorum fuga.",
     videoImage:
-      "https://i.ytimg.com/vi/JZk6T1TmIrs/hqdefault.jpg?sqp=-oaymwEbCKgBEF5IVfKriqkDDggBFQAAiEIYAXABwAEG&rs=AOn4CLAHAQWMJJRv2kmS32avbjO2bL3exw",
+      "https://i.ytimg.com/vi/Pvd1Y0OR2Nk/hqdefault.jpg?sqp=-oaymwEXCNACELwBSFryq4qpAwkIARUAAIhCGAE=&rs=AOn4CLCauoDvtkqaaSlaU5k8W1g-vA1hxQ",
+    views: "7",
+    videoCreated: "8",
+    videoUrl: "https://youtu.be/zj1X59IWLuQ",
+  },
+  {
+    _id: uuid(),
+    title: "Basics of CryptoCurrency",
```

Comment:
add tags for filter

Reply:
yep once completing p0 components will implement the filter feature